### DISCUSSION TITLE
Requesting custom touch controls for F1 23

### DISCUSSION THREAD

**Discussion by t00noob** [redphx/better-xcloud#317] [Custom touch controls]
Requested game: F1 23
Game link: https://www.xbox.com/en-GB/play/games/f1-23/9MTK0S8CQ0QV

Hey, I want to request a custom touch controls for the game, F1 23.

Down below is the mock-up for the touch controls :

![20240324_114144_0000](https://github.com/redphx/better-xcloud/assets/157677465/e3904945-3fe4-4fe6-b4b0-065e9c8d11db)

As you can see above, there's only a few things to be changed. This is due to the controls in F1 23 depends on the preset, the assist levels and also control preferences.

Most of the buttons such as Y & A has overlapping input, and the button B, X, LB and RB can be an input for other inputs if some control preferences were to be activated. This touch controls only useable if you use the preset number 1, since preset number 2 uses the left stick as the accelerate and brake button.

Here's the controls for the preset number 1:

![f1-23-1711250247392](https://github.com/redphx/better-xcloud/assets/157677465/6e21bd5a-4102-48ac-bb15-637091c290d7)
![f1-23-1711250277594](https://github.com/redphx/better-xcloud/assets/157677465/e2473406-dc18-4e39-bb05-a38571da1f79)

I'm still not fully understand on how F1 23 controls work, this is just a rough prototype of the touch controls. Feel free to learn more about the controls since I am kinda confused on how these things work.

**Comment by Tak-attack** [comment]
In the Forza layout, the accelerator pedal is set to have a handbrake button that is pressed when the button is "pulled".

In Preset1, the handbrake is set to Y, so it seems to me that this and the respective button icons should be changed.
